### DISCUSSION TITLE
Using acronyms.csv list

### DISCUSSION THREAD

**Discussion by briochemc** [retorquere/zotero-better-bibtex#2633] [Q&A]
I can't figure out how to use a list of acronyms. 

I want to use this for generating citation keys with abbreviated journal names. To do that I created a file `~/Zotero/better-bibtex/acronyms.csv` with the following content:
```
Earth and Planetary Science Letters, EPSL
"Earth and Planetary Science Letters", EPSL
EarthandPlanetaryScienceLetters, EPSL
```

And then in the citation key formula, I used
```
journal(abbrev="off").acronym
```

However, the generated citation key contains "EarthandPlanetaryScienceLetters" instead of the desired acronym, "EPSL". 

It seems none of the acronym lines work. What am I missing?

**Comment by retorquere** [comment]
I'd need a debug log to say.

**Comment by briochemc** [comment]
Sure, I think this is what you need? `4QUATXHP-refs-apse/6.7.114-6`

---
EDIT: Maybe you need the actual file here? In which case, here it is.

EDIT2: Removed the file because debug ID is preferred.

**Reply by retorquere** [reply]
The debug ID is preferred, my support tooling is built around it. I don't see an error in your log that would give an immediate indication of what is going wrong. Can you open a new issue and attach your acronym list there? You can use the existing debug log ID.
